index.js

```diff
@@ -141,8 +139,11 @@ module.exports = function create (opts) {
     }
 
     function showWindow (trayPos) {
-      var x = (opts.x !== undefined) ? opts.x : Math.floor(trayPos.x - ((opts.width / 2) || 200) + (trayPos.width / 2))
-      var y = (opts.y !== undefined) ? opts.y : trayPos.y
+      var position = positioner(opts['window-position'], [opts.width, opts.height], trayPos)
```

Comment:
do we wanna expose the positioner somehow so that it can be re-positioned later as the app is running?